### PR TITLE
docs: update 'ilab taxonomy diff' README section

### LINKED ISSUE #1910: README: consolidate the list and validate your new data commands to one
There are 2 steps listed in the list and validate your new data [section](https://github.com/instructlab/instructlab?tab=readme-ov-file#-list-and-validate-your-new-data), but they are basically the same thing. Consolidate to one command.

### PR DESCRIPTION
**Pull request by nathan-weinberg**
**Issue resolved by this Pull Request:**
Resolves #1910

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the
  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] [Changelog](https://github.com/instructlab/instructlab/blob/main/CHANGELOG.md) updated with breaking and/or notable changes for the next minor release.
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by alinaryan — APPROVED** [review]
Thanks for taking this on!

**Review by booxter — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -423,19 +423,13 @@ Detailed contribution instructions can be found in the [taxonomy repository](htt
 
 ### 📜 List and validate your new data
 
-1. List your new data by running the following command:
+To ensure `ilab` is registering your new knowledge or skills and that you've formatted them properly, you can run the `ilab taxonomy diff` command. This command will show new or modified YAML files within your taxonomy. For example, the following is the expected result after adding the new compositional skill `foo-lang` to the freeform writing subdirectory:
```
Comment (kelbrown20) [inline README.md:426]: Up to you! 
```suggestion
You can use the `ilab taxonomy diff` command to ensure `ilab` is registering your new knowledge or skills and your contributions are properly formatted. This command displays any new or modified YAML files within your taxonomy tree. For example, the following is the expected result of a valid compositional skill contribution after adding a new skill called `foo-lang.yaml` to the freeform writing subdirectory:
```

Comment (nathan-weinberg) [reply README.md:426]: Docs Queen :crown:

**Review by kelbrown20 — APPROVED** [review]

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**